flocker/provision/_sphinx.py

```diff
@@ -66,7 +71,11 @@ def run(self):
         task = getattr(tasks, 'task_%s' % (self.arguments[0],))
         prompt = self.options.get('prompt', '$')
         if len(self.arguments) > 1:
-            task_arguments = self.arguments[1].split()
+            # Some tasks can include the latest installable version as (part
+            # of) an argument. This replaces a placeholder with that version.
+            latest = get_installable_version(version)
+            task_arguments = [item.replace(PLACEHOLDER, latest) for
+                              item in self.arguments[1].split()]
```

Comment:
Can you please point to an example where this is done?  Why does the homebrew task get passed the argument, but does not use it?